frontend/src/app/component/dishes-component/dishes-component.css

```
@@ -0,0 +1,47 @@
+.dishes-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 100%;
+}
+
+.full-width-table {
+  width: 100%;
+  max-width: 1400px;
+  margin-bottom: 1.5rem; /* spațiu liber sub tabel */
```

```suggestion
  margin-bottom: 1.5rem;
```
we should just comments things that are not obvious, and the comments should be in english